src/apps/toolbar/modules/network/infra/WlanSelector.tsx

```diff
@@ -9,15 +9,21 @@ import { BackgroundByLayersV2 } from '../../../../seelenweg/components/Backgroun
 
 import { Selectors } from '../../shared/store/app';
 import { Icon } from 'src/apps/shared/components/Icon';
+import { cx } from 'src/apps/shared/styles';
 
 import { WlanBssEntry } from '../domain';
 
 import { WlanSelectorEntry } from './WlanSelectorEntry';
 
 function WlanSelector({ open }: { open: boolean }) {

```

Comment:
the code says: WLAN selector. so this shouldn't handle LAN connections. remove this code

Reply:
Program technically I do agree with with you, in case of UX, this cause problem in heads...
(When you have lan connection, why wlan component says it is connected to a WIFI, while the toolbar item shows it is connected to a LAN...)

Please think a bit in this way... I wait a bit after make the changes.

Reply:
because they are two separated network interfaces both can be connected, but the system will use one for internet based on speed.

Reply:
I know, you know, but what do you think everyone else know? :) That is the UX problem definition, when there is something which ment to be used somehow, but everyone who comes to that component awaits a bit different mechanism (cause of they used to, because it is natural that way...)

As I mentioned will remove the affected file if you insist. Anyway I found a minor performance issue, which I want to fix on the BE. There is no meaning to calculate IP change, when you have no internet. I would like to add that criteria to the else.

Reply:
remove it, another approach could be done, but this is not the correct one.

Reply:
removed